### PR TITLE
avoid loading vars on unspecified basedir (cwd)

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

vars manager
##### ANSIBLE VERSION

```
2.x
```

### CODE REVIEW

**Comment by abadger** [comment]
CC:  @misc

**Comment by abadger** [comment]
Timeout running pylint sanity test and what looks like unrelated transient issues on rhel/7.4/1  and windows/2012-R2/1  Hitting rebuild in shippable

**Comment by abadger** [comment]
windows/2012-R2/1 test is still failing in a way that is unrelated.

Please add a changelog and then merge and backport to all stable branches back to 2.4.

File: changelogs/fragments/avoid_cwd_vars.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - avoid loading host/group vars from cwd when not specifying a playbook or playbook base dir
```
Comment (abadger) [inline changelogs/fragments/avoid_cwd_vars.yml]: We don't yet have a security category, so please prepend ```**Security Fix** -``` to this:

```
bugfixes:
    - ** Security Fix ** - avoid loading host/group vars from cwd when not specifying a playbook or playbook base dir
```

**Comment by abadger** [comment]
The following failed but look like they are unrelated:
* T=windows/2012-R2/1 
* T=rhel/7.4/1

**Comment by bcoca** [comment]
fix for CVE-2018-10874

**Pull request merged**